Source/Core/StyleSheetParser.h

```diff
@@ -37,11 +37,12 @@ namespace Rml {
 class PropertyDictionary;
 class Stream;
 class StyleSheetContainer;
+struct MediaBlock;
 class StyleSheetNode;
 class AbstractPropertyParser;
 struct PropertySource;
 using StyleSheetNodeListRaw = Vector<StyleSheetNode*>;
-using MediaBlockListRaw = Vector<Pair<PropertyDictionary, UniquePtr<StyleSheet>>>;
+using MediaBlockListRaw = Vector<MediaBlock>;
```

Comment:
```suggestion
using MediaBlockList = Vector<MediaBlock>;
```
The raw suffix on the previous type actually referred to the contained raw pointer, since StyleSheetNodeList is already used as container of unique pointers. That's not the case here, so we can drop that suffix. Totally understand the confusion here though :)